### ISSUE TITLE
OutOfMemory when uploading image

### ISSUE THREAD

**Issue by padyx** [padyx/purplemoon-android#17]
Crash when uploading image - not enough memory.

Application version   
    1.0.0

Android version
    Android 4.0.3 - 4.0.4

Device    
    Galaxy Tab 8.9 (GT-P7310)

```
#!java

java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:278)
at java.util.concurrent.FutureTask$Sync.innerSetException(FutureTask.java:273)
at java.util.concurrent.FutureTask.setException(FutureTask.java:124)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:307)
at java.util.concurrent.FutureTask.run(FutureTask.java:137)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:208)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
at java.lang.Thread.run(Thread.java:856)
Caused by: java.lang.OutOfMemoryError
at android.graphics.BitmapFactory.nativeDecodeStream(Native Method)
at android.graphics.BitmapFactory.decodeStream(BitmapFactory.java:584)
at android.graphics.BitmapFactory.decodeStream(BitmapFactory.java:640)
at ch.defiant.purplesky.dialogs.UploadPhotoDialogFragment$PictureLoaderTask.doInBackground(UploadPhotoDialogFragment.java:192)
at ch.defiant.purplesky.dialogs.UploadPhotoDialogFragment$PictureLoaderTask.doInBackground(UploadPhotoDialogFragment.java:1)
at android.os.AsyncTask$2.call(AsyncTask.java:264)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:305)
... 5 more
```

**Comment by padyx** [comment]
(Original author of comment is Bitbucket user 'Chakotay')
FIxed in head

**Comment by padyx** [comment]
(Original author of comment is Bitbucket user 'Chakotay')
Fix bug #95: OutOfMemory when loading image for upload


→ <<cset d788d7b30d99>>
